resources/content/schema/admin/admin-auth.json

```diff
@@ -97,6 +97,8 @@
         "processDefinition.resourceType" : "r",
         "processDefinition.stateTransitions" : "r",
 
+        "stateTransition" : "r",
```

Comment:
Craig, should we also set the R access on the stateTransition object's fields? like we do for other resources. Or for this particular use case, setting access rights on object itself is enough?